dev/tasks/exec/tasks.js

```diff
@@ -0,0 +1,53 @@
+/**
+ * @license Copyright (c) 2003-2016, CKSource - Frederico Knabben. All rights reserved.
+ * For licensing, see LICENSE.md.
+ */
+
+'use strict';
+
+const gulp = require( 'gulp' );
+const gutil = require( 'gulp-util' );
+const minimist = require( 'minimist' );
+
+const exec = require( './tasks/exec' );
+const log = require( '../../lib/log' );
+
+module.exports = ( config ) => {
+	const ckeditor5Path = process.cwd();
+	const packageJSON = require( '../../../package.json' );
+
+	// Configure logging.
+	log.configure(
```

Comment:
The `ckeditor5/dev/lib/log` module is used only in `ckeditor5/dev/tasks/dev/tasks.js` and `ckeditor5/dev/tasks/exec/tasks.js`. Seems like it is close copy of `ckeditor5-utils/log`.
Having that in mind we could consider removing all that "configuration" and hardcode it into `ckeditor5/dev/lib/log`. What is your opinion about this?